README.md

```diff
@@ -10,6 +10,34 @@ on the provided pseudo-random seed.
 Each privileged application has its own sortition pool and is responsible for
 maintaining operator weights up-to-date.
 
+## Setup
```

Comment:
I know it was not part of this change but in the paragraph above it says
> In the Keep network the stake consists of staked KEEP tokens.

Is this relevant anymore? Shouldn't it be Threshold and T now?